### PR TITLE
Fixing error response for CosmosDB 2020-06-01-preview

### PR DESCRIPTION
**Pull request by nichatur**
# All SDK Contribution checklist:

Swagger PR: https://github.com/Azure/azure-rest-api-specs/pull/12800

This checklist is used to make sure that common guidelines for a pull request are followed.
- [ ] **Please open PR in `Draft` mode if it is:**
	- Work in progress or not intended to be merged.
	- Encountering multiple pipeline failures and working on fixes.
- [ x If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/breaking-change-rules.md).**

### [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code. (Track 2 only)
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK. Please double check nuget.org current release version.

## Additional management plane SDK specific contribution checklist: 
Note: Only applies to `Microsoft.Azure.Management.[RP]` or `Azure.ResourceManager.[RP]`
 
- [x] Include updated [management metadata](https://github.com/Azure/azure-sdk-for-net/tree/master/eng/mgmt/mgmtmetadata).
- [x] Update AzureRP.props to add/remove version info to maintain up to date API versions.

### Management plane SDK Troubleshooting
- If this is very first SDK for a services and you are adding new service folders directly under /SDK, please add `new service` label and/or contact assigned reviewer.
- If the check fails at the `Verify Code Generation` step, please ensure:
	- Do not modify any code in generated folders.
	- Do not selectively include/remove generated files in the PR.
	- Do use `generate.ps1/cmd` to generate this PR instead of calling `autorest` directly.
	Please pay attention to the @microsoft.csharp version output after running generate.ps1. If it is lower than current released version (2.3.82), please run it again as it should pull down the latest version,

### Old outstanding PR cleanup
 Please note:
	If PRs (including draft) has been out for more than 60 days and there are no responses from our query or followups, they will be closed to maintain a concise list for our reviewers.

### CODE REVIEW

**Comment by YalinLi0312** [comment]
@nichatur Could you fix the failing ci tests?

File: sdk/cosmosdb/Microsoft.Azure.Management.CosmosDB/src/Microsoft.Azure.Management.CosmosDB.csproj [hunk 1]
```diff
@@ -6,18 +6,12 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.CosmosDB</PackageId>
     <Description>Provides developers with libraries for the CosmosDB under Azure Resource manager to manage CosmosDB Account, Databases and child resources and their available management capabilities. Create, Delete, Update CosmosDB Account, Databases and child resources. Note: This client library is for CosmosDB under Azure Resource Manager.</Description>
-    <Version>1.4.0-preview</Version>
+    <Version>1.5.0-preview</Version>
```
Comment (MehaKaushik) [inline sdk/cosmosdb/Microsoft.Azure.Management.CosmosDB/src/Microsoft.Azure.Management.CosmosDB.csproj:9]: Lets name this 1.4.1-preview.

File: sdk/cosmosdb/Microsoft.Azure.Management.CosmosDB/src/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure CosmosDB Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure CosmosDB Resources.")]
 
-[assembly: AssemblyVersion("1.3.3.0")]
-[assembly: AssemblyFileVersion("1.3.3.0")]
+[assembly: AssemblyVersion("1.3.5.0")]
+[assembly: AssemblyFileVersion("1.3.5.0")]
```
Comment (MehaKaushik) [inline sdk/cosmosdb/Microsoft.Azure.Management.CosmosDB/src/Properties/AssemblyInfo.cs:11]: This version should be the same as the version in CosmosDB.scproj.  (1.4.1.0 here )

**Comment by MehaKaushik** [comment]
Hey @YalinLi0312, the ci is failing bcz of discrepancy in generated code and reference. The discrepancy arise bcz we are using preview version to generate this code and our swagger specs define stable version as the default tag. In this PR we intend to check in and release the preview apiVersion. We don't want to change our default version in swagger just to get this checked in. 

Please help us merge this PR in its current state.

**Review by allenjzhang — APPROVED** [review]

**Comment by allenjzhang** [comment]
Reviewed code generation diff. It is correct. Overriding the error.

**Pull request merged**